modules/sfm/CMakeLists.txt

```diff
@@ -154,7 +154,7 @@ endif()
 #include_directories(${OCV_TARGET_INCLUDE_DIRS_${the_module}})
 add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/src/libmv_light" "${CMAKE_CURRENT_BINARY_DIR}/src/libmv")
 
-ocv_target_link_libraries(${the_module} ${LIBMV_LIGHT_LIBS})
+ocv_target_link_libraries(${the_module} ${LIBMV_LIGHT_LIBS} opencv_imgcodecs)
```

Comment:
On the line 114 there is already included `opencv_imgcodecs` dependency of the module. No need to duplicate that.

More details are required:
- failed command line via `make VERBOSE=1` (without any `-j` option)

Reply:
Perhaps we need dependency for libmv library instead of opencv sfm module.

Try to modify this line instead:
https://github.com/opencv/opencv_contrib/blob/4.5.3/modules/sfm/CMakeLists.txt#L98